### PR TITLE
refactored using __init__ and class import for #83

### PR DESCRIPTION
**Pull request by kgashok**
Partial fix for #83 
 - TIL `AttributeError: Module "foo" has no attribute "bar"` has a easy resolution
 - Thanks to [this (a hack?)](https://containersolutions.github.io/runbooks/posts/python/module-has-no-attribute/#step-2), os/sys calls have been refactored out. 

Check everything which applies

- [x] I have added the issue number for which this pull request is created.
- [x] Ran `pytest` locally to completion.

### CODE REVIEW

File: tests/test_basics.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
-import os
-import sys
-relpath = os.path.dirname(__file__) # Find relative path: from current directory to test.py
-abspath = os.path.abspath(".")      # Find absolute path: from / to current directory
-os.chdir(abspath + '/first_timers')
-sys.path.append('./')
-import first_timers
+# import os
```
Comment (arshadkazmi42) [inline tests/test_basics.py:1]: Can we remove these comments.

As we are doing cleanups, I think we should remove these commented code which we are not using.

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thanks for this, using `Makefile` seems like a good idea.

I have added a small comment, let me know your thoughts

File: tests/test_basics.py (hunk 1 above)
Comment (kgashok) [reply tests/test_basics.py:1]: I left them in there, so it is easy for you to revert back in case you changed your mind. 
I will remove them now.

Comment (kgashok) [reply tests/test_basics.py]: @arshadkazmi42 clean up done.

**Review by arshadkazmi42 — APPROVED** [review]
:100:

**Review by BadKarma187 — COMMENTED** [review]
good

**Pull request merged**